samples/csharp/getting-started/Ranking_PersonalizedSort/PersonalizedRanking/Program.cs

```diff
@@ -0,0 +1,189 @@
+            IEstimator<ITransformer> trainer = mlContext.Ranking.Trainers.LightGbm(options);
+            IEstimator<ITransformer> trainerPipeline = dataPipeline.Append(trainer);
+
+            Console.WriteLine("===== Train the model =====");
+
+            // Training the model is a process of running the chosen algorithm on the given data. To perform training you need to call the Fit() method.
+            ITransformer model = trainerPipeline.Fit(trainData);
+
+            Console.WriteLine("===== Save the model =====");
+
+            // Save the model
+            mlContext.Model.Save(model, trainData.Schema, modelPath);
+
+            return model;
+        }
+
+        static void EvaluateModel(MLContext mlContext, ITransformer model, string testDatasetPath)
+        {
+            Console.WriteLine("===== Evaluate the model's accuracy with test data =====");
+
+            // Load the test data and use the model to perform predictions on the test data.
+            IDataView testData = mlContext.Data.LoadFromTextFile<HotelData>(testDatasetPath, separatorChar: ',', hasHeader: true);
+            IDataView predictions = model.Transform(testData);
+
+            // Evaluate the metrics for the data using NDCG; by default, metrics for the up to 3 search results in the query are reported (e.g. NDCG@3).
+            ConsoleHelper.EvaluateMetrics(mlContext, predictions);
+
+            // Evaluate metrics for up to 10 search results (e.g. NDCG@10).
+            ConsoleHelper.EvaluateMetrics(mlContext, predictions, 10);
+        }
+
+        public static void ConsumeModel(MLContext mlContext, string modelPath, string exampleDatasetPath)
+        {
+            Console.WriteLine("===== Consume the model =====");
+
+            DataViewSchema predictionPipelineSchema;
+            ITransformer predictionPipeline = mlContext.Model.Load(modelPath, out predictionPipelineSchema);
+
+            // Load example data and use the model to perform predictions on it.
+            IDataView exampleData = mlContext.Data.LoadFromTextFile<HotelData>(exampleDatasetPath, separatorChar: ',', hasHeader: true);
+
+            // Predict rankings.
+            IDataView predictions = predictionPipeline.Transform(exampleData);
+
+            // In the predictions, get the scores of the hotel search results included in the first query (e.g. group).
+            IEnumerable<HotelPrediction> hotelQueries = mlContext.Data.CreateEnumerable<HotelPrediction>(predictions, reuseRowObject: false);
+            var firstGroupId = hotelQueries.First<HotelPrediction>().GroupId;
+            IEnumerable<HotelPrediction> firstGroupPredictions = hotelQueries.Take(50).Where(p => p.GroupId == firstGroupId).OrderByDescending(p => p.PredictedRank).ToList();
+
+            // The individual scores themselves are NOT a useful measure of accuracy; instead, they are used to determine the ranking where a higher score indicates a higher ranking.
```

Comment:
I'd avoid the term `accuracy` when discussing metrics, unless you're specifically talking about the `accuracy metric`:
```suggestion
            // The individual scores themselves are NOT a useful measure of result quality; they are only useful as a relative measure to other scores in the group. The scores are used to determine the ranking where a higher score indicates a higher ranking vs another result.
```

Reply:
I also replaced other references to "accuracy" with "quality".